content/technotes/2020-07-07-github-pat-credentials.md

```diff
@@ -0,0 +1,99 @@
+---
+slug: github-pat
+title: 'A better way to manage your Github personal access tokens'
+date: '2020-07-07'
+author:
+  - Jeroen Ooms
+tags:
+  - R
+  - package
+  - gert
+  - credentials
+  - git
+  - github
+---
+
+We have been working hard behind the scenes on the upcoming release of our new git package named [gert](https://docs.ropensci.org/gert), a joint effort from rOpenSci and the Tidyverse team. One of the main features of gert is the out-of-the-box authentication mechanism, which is provided via the new [credentials](https://docs.ropensci.org/credentials/articles/intro.html) package.
```

Comment:
Nitpick: I write about Git a lot too in usethis and have decided to try to say capital-G "Git" because that seems to be the official capitalization. I also acknowledge this is utterly meaningless. I won't bring it up again.